pages/about.html

```diff
@@ -4,9 +4,44 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/about.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header> 
+        <nav>
+          <a href="/index.html">Home</a>
+          <a href="/pages/portfolio.html">Portfolio</a>
+          <a href="/pages/about.html">About</a>
+          </nav>
+      </header> 
+  <main>
+  <h1>ABOUT ME</h1>
+    
+      Hello, my name is cristal and I am in a student at ADA Developers Academy. 
+      In my free time, I enjoy spending time with my family and jogging. 
+      I am from Washington State and I love living here.
+        <p>
+        I began programming and learning python for fun during my senior year of high 
+        school but I never considered that it could be a possible career for me. I thought this 
+        because I did not think there was a space for me in the field and I had some doubts. I went 
+        on to major in Applied Developmental Psychology and Spanish and graduated from Eastern Washington 
+        University. During quarantine, which was during my final quarter of undergrad, I began to explore 
+        different careers and I stumbled upon software development once more. This time, I did more research 
+        and continued to learn. I read about Ada Developers Academy and it was then that I thought about the 
+        possibility of me actually pursuing this as a career.
+      </p>
+      <p>
+        <img src="/images/wa.png" alt="WA state">
+        <img src="/images/ada1.png" alt="ada" >
```

Comment:
To avoidabsolute paths here we would write
```html
        <img src="../images/wa.png" alt="WA state">
        <img src="../images/ada1.png" alt="ada" >
```

We need to move up a direcotry `../` and then from there look in the images folder `images/`